model/job_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
```

Comment:
What are these for? Seems unrelated?

Reply:
I can put them into a separate PR. I originally put the `HandleJobPanic` function in `model/job.go` and noticed that there were no tests for the file. So when writing the test I also tested the rest of the methods as well.

Reply:
Yeah let's keep this PR scoped to only this task :)